linuxkit/linuxkit
# runc: bump to newest version

This version includes a fix for CVE-2024-21626 which allowed an attacker in bad circumstances to
"escape containerized environments".

See also https://access.redhat.com/security/cve/cve-2024-21626



**- What I did**

Bumped runc version

**- How I did it**
Changed variable in dockerfile

**- How to verify it**
1. Remove in runc/Dockerfile everything from '#FROM scratch' (including itself)
2. Run 'docker build -t runc . && docker run -it runc runc --version"
3. Check that the version if 1.1.12

**- Description for the changelog**
Bumping runc to fix CVE-2024-21626


**- A picture of a cute animal (not mandatory but encouraged)**

https://en.wikipedia.org/wiki/Punxsutawney_Phil#/media/File:Punxsutawney_Phil_2018_(cropped).jpg
